conanfile.py

```diff
@@ -208,6 +208,7 @@ def _generate_pyinstaller_spec(self, location, entrypoint_location, icon_path, e
             else:
                 continue
             if not src_path.exists():
+                self.output.warning(f"Source path for binary {binary['binary']} does not exist")
                 continue
             for bin in src_path.glob(binary["binary"] + ".*[exe|dll|so|dylib]"):
```

Comment:
Since it also complains about *.so.1` this glob statement should probably be something like:
```suggestion
            for bin in src_path.glob(binary["binary"] + ".*[exe|dll|so|dylib]*"):
```